src/main/kotlin/com/sourcegraph/cody/config/SettingsMigration.kt

```diff
@@ -99,32 +99,30 @@ class SettingsMigration : Activity {
   }
 
   private fun migrateLlms(project: Project) {
```

Comment:
I think that won't work. Past migrations run only once.
If you need to migrate something now you need to add completely new function.
Also this way migrations are linear, changing past migrations is a bit more risky as we can accidentally mess up something in the chain of changes.

Reply:
true ☝️

Reply:
Done